pkg-manifest/exportable-manifest/test/index.test.ts

```diff
@@ -106,5 +111,10 @@ test('workspace deps are replaced', async () => {
       baz: '^1.2.3',
       foo: '4.5.6',
     },
+    peerDependencies: {
+      bar: 'npm:@foo/bar@>3.2.1',
```

Comment:
May of done this one wrong. I haven't seen this specifier before. Does having `<package>@<range specifier><verison>`?

Perhaps one would only ever do `<package>@<version>`

Reply:
This doesn't make sense in peer dependencies. There is no point in using an alias in peer deps.